spring-integration-websocket/src/main/java/org/springframework/integration/websocket/ServerWebSocketContainer.java

```diff
@@ -56,20 +59,22 @@ public class ServerWebSocketContainer extends IntegrationWebSocketContainer
 
 	private final String[] paths;
 
+	@SuppressWarnings("NullAway.Init")
```

Comment:
This can be null if respective setter is not called.
Let's have `Assert.notNull()` before calling `.setHandshakeHandler(this.handshakeHandler)` in the `registerWebSocketHandlers()`!